ios/RPCModule.swift

```diff
@@ -123,21 +113,10 @@ class RPCModule: NSObject {
     }
   }
 
-  // new way to read the wallet file -> Decoded Data
-  func readWallet() throws -> Data {
-    let fileName = try getFileName(walletFileName)
-    do {
-      let content = try Data(contentsOf: URL(fileURLWithPath: fileName))
-      return content
-    } catch {
-      throw FileError.readWalletDecodedDataError("Error reading new wallet format error: \(error.localizedDescription)")
-    }
-  }
-
-  func readWalletBackup() throws -> Data {
+  func readWalletBackup() throws -> String {
     let fileName = try getFileName(walletBackupFileName)
     do {
-      let content = try Data(contentsOf: URL(fileURLWithPath: fileName))
+      let content = try String(contentsOfFile: fileName, encoding: .utf8)
```

Comment:
I don't understand why we are specifying `utf8` encoding here.   I feel confident that the only thing in the wallet file that *might* be utf8 encoded is the memo.   In general the wallet file should be binary data...  I am fairly sure.   @AloeareV @fluidvanadium ?

Reply:
Well, since the problem we have is not related with uft8 representation, the most cautious move is to revert it to the prior situation and think about it for a while, because if we change the internal format of the wallet file, we need to try reading the content with two different methods (try & error) and I don't like this situation.
I want to think for a while to find the better way to do this disrupting change. But I'll do, because I agree that we don't need to specify uft8, binary data is the right thing to do here.

Reply:
I'm going to do ASAP in a separate PR for next version.